package.json

```diff
@@ -126,10 +127,12 @@
     "vanilla-framework": "^3.15.1 || ^4.0.0"
   },
   "scripts": {
-    "build": "rm -rf dist && yarn build-local; yarn build-declaration",
-    "build-local": "NODE_ENV=production babel src --out-dir dist --copy-files --extensions '.js,.jsx,.ts,.tsx'",
+    "build": "rm -rf dist && yarn build-cjs && yarn build-declaration && yarn build-esm && yarn build-declaration-esm",
+    "build-cjs": "NODE_ENV=production BABEL_ENV=cjs babel src --out-dir dist --copy-files --extensions '.js,.jsx,.ts,.tsx'",
```

Comment:
I understand your point, and I agree with you. Having both types separated into two different directories is much cleaner.
The reason for leaving the `cjs` version as it is, is mainly to avoid breaking existing projects that are using this package with absolute paths to components. I'm sure that these imports aren't needed as everything should be importable from `/dist/cjs/index.js`. 
Here is a set of projects that have this problem, this list is probably only a subset of the total projects that have this issue: https://github.com/search?q=org%3Acanonical+canonical%2Freact-components%2Fdist&type=code